tests/test_adjoint_jacobian.py

```diff
@@ -605,7 +641,9 @@ def test_state_return_type(self, dev):
 class TestAdjointJacobianQNode:
     """Test QNode integration with the adjoint_jacobian method"""
 
-    @pytest.fixture(params=[np.complex64, np.complex128])
+    @pytest.fixture(
+        params=[np.complex64, np.complex128] if device_name != "lightning.gpu" else [np.complex128]
```

Comment:
We should be able to handle FP32 no? I assumed we have compiled all backends to support this, even if some of the object need to be returned as double?

What happens if we use np.complex64 here?

Reply:
Fixed